astronomer/providers/core/example_dags/example_external_deployment_trigger_dag_operator.py

```diff
@@ -0,0 +1,82 @@
+import os
+from datetime import timedelta
+from typing import Any, Dict
+
+import requests
+from airflow import DAG
+from airflow.hooks.base import BaseHook
+from airflow.operators.python import PythonOperator
+from airflow.utils.log.secrets_masker import mask_secret
+from airflow.utils.timezone import datetime
+
+from astronomer.providers.core.operators.external_trigger_dagrun import (
+    ExternalDeploymentTriggerDagRunOperator,
+)
+
+DEPLOYMENT_CONN_ID = os.getenv("ASTRO_DEPLOYMENT_CONN_ID", "")
+ASTRO_AUTH_CONN_ID = os.getenv("ASTRO_AUTH_CONN_ID", "")
```

Comment:
If we have one, can we please add a link to the documentation in a comment here describing how to setup an Astro auth connection?